### PR TITLE
Change p10 to p9

### PR DESCRIPTION
**Pull request by Neha-dot-Yadav**
This PR updates the hardware version from p10 to p9 for powervm in 4.13 pipelines.
Signed-off-by: Neha Yadav <neha.yadav3@ibm.com>

### CODE REVIEW

File: jobs/pipelines/powervm/odf/4.12/daily-odf4.12-powervm-p9-tier-2 [hunk 1]
```diff
@@ -201,5 +201,3 @@ pipeline {
         }
     }
 }
-
```
Comment (Shilpi-Das1) [inline jobs/pipelines/powervm/odf/4.12/daily-odf4.12-powervm-p9-tier-2:204]: @Neha-dot-Yadav You forgot do to change here

Comment (Neha-dot-Yadav) [reply jobs/pipelines/powervm/odf/4.12/daily-odf4.12-powervm-p9-tier-2]: @Shilpi-Das1  updated it

**Review by Shilpi-Das1 — APPROVED** [review]
/lgtm

**Review by sudeeshjohn — APPROVED** [review]
/lgtm

**Pull request merged**